localstack/services/dynamodb/dynamodb_listener.py

```diff
@@ -152,13 +153,19 @@ def forward_request(self, method, path, data, headers):
 
         elif action == 'BatchWriteItem':
             existing_items = []
+            unprocessed_items = []
             for table_name in sorted(data['RequestItems'].keys()):
                 for request in data['RequestItems'][table_name]:
                     for key in ['PutRequest', 'DeleteRequest']:
                         inner_request = request.get(key)
                         if inner_request:
-                            existing_items.append(find_existing_item(inner_request, table_name))
+                            if self.should_throttle(action):
+                                unprocessed_items.append(inner_request)
+                            else:
+                                item = find_existing_item(inner_request, table_name)
+                                existing_items.append(item)
             ProxyListenerDynamoDB.thread_local.existing_items = existing_items
+            ProxyListenerDynamoDB.thread_local.unprocessed_items = unprocessed_items
```

Comment:
Suggestion:

Split `unprocessed_items` into `unprocessed_put_items` and `unprocessed_delete_items` to make it easier to distribute amongst the response `dict`